src/main/scala/org/broadinstitute/hail/driver/IBDCommand.scala

```diff
@@ -52,15 +60,26 @@ object IBDCommand extends Command {
   }
 
   override protected def run(state: State, options: Options): State = {
-    val computeMaf = Option(options.computeMafExpr).map(generateComputeMaf(state.vds, _))
+    val min = if (options.min == -1) None else Some(options.min)
+    val max = if (options.max == -1) None else Some(options.max)
+
+    min.foreach(min => optionCheckInRangeInclusive(0.0, 1.0)("minimum", min))
+    max.foreach(max => optionCheckInRangeInclusive(0.0, 1.0)("maximum", max))
 
-    val ibdMap = IBD(state.vds, computeMaf, !options.unbounded)
+    min.foreach(min => max.foreach(max =>
```

Comment:
can use the "liftedZip" utility function to go from Option[A], Option[B] to Option[(A, V)]